core/templates/pages/email-dashboard-pages/email-dashboard-result.component.spec.ts

```diff
@@ -76,7 +77,7 @@ describe('Email Dashboard Result Page', function() {
     expect(ctrl.testEmailSentSuccesfully).toBe(false);
   });
 
-  it('should sucessfully submit email', function() {
+  it('should send email when submitting email', function() {
```

Comment:
should submit an email when fields are populated with valid values.

Reply:
Done.